ruby-challenges.rb

```diff
@@ -3,32 +3,85 @@
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
+#Psuedo Code:
+#1. Create a method called is_even_or_odd that takes in an integer as an input
+#2. use a conditional statement to check if the int is either even or odd
+#3. use the method .even? to perform the check
+#4. can use a simple else statement as a catchall
+#5. print the output string -> "#{num1} is odd" or "#{num1} is even"
+
 num1 = 7
 # Expected output: '7 is odd'
 num2 = 42
 # Expected output: '42 is even'
 num3 = 221
 # Expected output: '221 is odd'
 
+def is_even_or_odd num
+    if num.even?
+        "#{num} is even"
+    else
+        "#{num} is odd"
+    end
+end
+
+p is_even_or_odd num1
+p is_even_or_odd num2
+p is_even_or_odd num3
 
 
 # -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. 
 # HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
 
+#Psuedo Code
+#1. create a method called vowel_removal that takes a string as input
+#2. create a new variable to store the new string in for output
+#3. use the built in method .delete to remove vowels from the given string by giving the argument of all vowels -> "aeiou"
+#4. return the variable used to store the new string
+
 beatles_album1 = 'Rubber Soul'
 # Expected output: 'Rbbr Sl'
 beatles_album2 = 'Sgt Pepper'
 # Expected output: 'Sgt Pppr'
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def vowel_removal string
+    caps_check_string = string.delete('AEIOU')
+    downcase_string_check = caps_check_string.delete('aeiou')
+    new_string = downcase_string_check
+   p new_string 
+end
+vowel_removal beatles_album1
+vowel_removal beatles_album2
+vowel_removal beatles_album3
+
```

Comment:
cool. now that the code works, go back and see where you can reduce the lines of code. Maybe you can delete the upper and lowercase vowels at the same time?